src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/WhereCQLClauseAnalyzer.java

```diff
@@ -70,6 +70,8 @@ AnalysisStrategy getStrategy(WhereCQLClauseAnalyzer analyzer) {
                     analyzer::checkNoFilters,
                     analyzer::checkAllColumnsExist,
                     analyzer::checkNonPrimaryKeyFilters,
+                    analyzer::checkOnlyEqUsage,
+                    // TODO, checkOnlyEqUsage rule make checkNoInFilterUsage redundant.
```

Comment:
Not sure I understand this comment; is it obsolete, or should it be rewritten?

Reply:
agree, remove the $in check if not needed any more and remove the exception

Reply:
removed.